packages/react/src/components/TableCard/TableCard.jsx

```diff
@@ -453,9 +452,7 @@ const TableCard = ({
         : hasActionColumn || filteredPrecisionColumns.length || thresholds
         ? tableData.map((i) => {
             // if has custom action
-            const action = hasActionColumn
-              ? { actionColumn: JSON.stringify(i.actions || []) }
```

Comment:
The source of the original stringify of action items. Anyone know why this was done?

Reply:
I don't know. Perhaps to create a deep clone? Are they modified at some point later?